database/awsImageSeeding.js

```diff
@@ -0,0 +1,67 @@
+const AWS = require('aws-sdk');
+const faker = require('faker');
+const fetch = require('node-fetch');
```

Comment:
why did you decide to use fetch vs axios??

Reply:
great question. I mostly chose fetch becuase I say an example of someusing fetch instead of axios and I was too lazy/it didnt occur to me to use axios instead... lol